I14467 update canton overview docs (#444)
digital-asset/docs.daml.com: docs/2.8.0/docs/canton/architecture/overview.rst

of the transaction).

There is only a bounded relationship between these times, ensuring that the `ledger time` must be
in a pre-defined bound around the `record time`. The tolerance is defined on the domain

> as a domain parameter, known to all participants

Reviewer: ```suggestion
as a domain parameter, known to all participants:
```

**Ordering:** The sequencer provides a `global total-order
multicast <http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.85.3282&rep=rep1&type=pdf>`__

> where envelopes are uniquely time-stamped and the global ordering is

Reviewer: ```suggestion
where envelopes are uniquely timestamped and the global ordering is
```

derived from the timestamps. Instead of delivering a single envelope, the
sequencer provides batching, that is, a

> list of individual envelopes are submitted. All these envelopes get the

Reviewer: ```suggestion
list of individual envelopes are submitted. All of these envelopes get the
```

timestamp of the batch they are contained in. Each envelope may
have a different set of recipients; the envelopes in each recipient's batch
are in the same order as in the sent batch.

**Evidence:** The sequencer provides the recipients with a
cryptographic proof of authenticity for every batch it
delivers, including evidence on the order of envelopes.

**Sender and Recipient Privacy:** The recipients
do not learn the identity of the submitting participant.
A recipient only learns the identities of recipients
on a particular envelope from a batch if it is itself a recipient of that
envelope.

.. note:: In the implementation, the recipients of an envelope are not a set of members (as indicated above),
   but a forest of sets of members.
   A member receives an envelope if it appears somewhere in the recipient forest.

>    A member sees the nodes of the forest that contain itself as a recipient as well as all descendants of such nodes;

Reviewer: ```suggestion
   A member sees the nodes of the forest that contain itself as a recipient (as well as all descendants of such nodes),
```

by not revealing their identities to each other.
At a high level, the mediator:

- collects mediator responses from participants,
- validates them according to the Canton protocol,
- computes the mediator verdict (approve / reject / timed out) according to the confirmation policy, and
- sends the result message.

Additionally, for auditability the mediator persists every finalized request

> together with its verdict in a long term storage

Reviewer: ```suggestion
together with its verdict in long-term storage
```

Participant-internal Canton Components
--------------------------------------

The below diagram shows the main components of a participant.

> .. https://lucid.app/lucidchart/effe03fe-e952-49e2-abc7-ac7a4cc56211/edit?page=0_0&invitationId=inv_f3433f52-c2ec-4854-9b36-58942657f191#

Reviewer: whoops, misunderstood. Resolving

.. image:: ./images/overview/canton-participant-components.svg
   :align: center
   :width: 100%

> A ledger application uses the ledger API (at the top of the diagram) to send commands to a participant and

Author: done

to receive the corresponding events.
The command submission service receives a command, parses it, and performs some basic validation.
Next, the command is submitted to DAMLe (DAML engine), which translates it to a transaction;
a command consists only of a root node whereas a transaction also recursively contains all consequences of all exercise actions.
Then, the domain router chooses a domain that is suitable for executing the transaction.

The transaction processor translates the transaction to a confirmation request;
in particular, it computes the view decomposition, embeds the transaction into a Merkle tree, and
creates different envelopes tailored to the different members validating the request.
It uses the sequencer client to send the confirmation request to the mediator and all participants involved in the transaction.

The transaction processor also uses the sequencer client to receive confirmation requests from the domain,
to send mediator responses, and to receive the result messages from the mediator.

> The multi domain event log stores for every request if has been committed vs. rolled back.

Reviewer: Words missing here? Stores *what* for every request? "if has been committed" -- if *it* has been committed? 

Also "multi domain" needs a hyphen: "multi-domain"

Maybe it's this:
```suggestion
The multi-domain event log stores every committed request (but not rolled-back requests).
```

Author: I couldn't apply this as is, because is also stores rollbacks.

Reviewer: Yep, I said "maybe" because I wasn't sure I understood... 🙂 Updated my suggestion.
